projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -56,6 +56,14 @@
     }
   }
 
+  .sidebar {
```

Comment:
I'm wondering if that is correct place for these styles. In html for this component I can't see that this class is used. Also note that this component has encapsulation: ViewEncapsulation.None, so having this class not nested in scss may cause that you affect other places with sidebar class too. @MichalKinas what do you think?

Reply:
Yeah I agree, component itself shouldn't style elements that it's not aware of. In this case element containing sidebar class comes from ng-content

Reply:
This was removed in a previous commit, I only identified what got removed and placed it back.

Reply:
discussed on slack, please leave and for now and raise jira as suggested by @MichalKinas